MdeModulePkg/Universal/Console/ConPlatformDxe/ConPlatform.c

```diff
@@ -809,7 +809,7 @@ MatchUsbClass (
   DeviceSubClass = DevDesc.DeviceSubClass;
   DeviceProtocol = DevDesc.DeviceProtocol;
 
-  // MSCHANGE - BEGIN - Add support for IAD-type multi-function devices.

```

Comment:
Where was this one checked in? Should we merge this change into the same commit?

Reply:
I added the suggested MS_CHANGE to MU_CHANGE to other places. But I think we should drop this one as it is a comment only change and not needed to be added as a separate commit.